boards/seeedstudio-gd32/Makefile.include

```diff
@@ -0,0 +1,11 @@
+# set default port depending on operating system
+PORT_LINUX ?= /dev/ttyUSB1
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+
+# setup serial terminal
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+#OPENOCD_CMD_RESET_RUN =-c _reset
+include $(RIOTMAKE)/tools/openocd.inc.mk
```

Comment:
And this, just specify `PROGRAMMER ?= openocd` instead.

Reply:
Thanks!